packages/protocol/test/governance/validators.ts

```diff
@@ -2184,4 +2184,129 @@ contract('Validators', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#groupMembershipInEpoch', async () => {
+    const validator = accounts[0]
+    const groups = accounts.slice(1, -1)
+
+    beforeEach(async () => {
```

Comment:
Not needed

Reply:
I added `await`s before `setAddressFor` to silence linter complaints, so now it is

Reply:
No I mean there's no need to call `setAddressFor` for this test since `groupMembershipInEpoch` can be called by anyone